### ISSUE TITLE
Plugin can't parse relative paths anymore in "use"

### ISSUE THREAD

**Issue by e-oz** [intellij-rust/intellij-rust#1835]
Couple updates ago all "use" statements with leading "::" stopped working, so currently I'm not sure why I still use this plugin because refactoring doesn't work, cmd+click doesn't work and I see a lot of false error warnings (half of the code is red in IDE when it compiles without a single warning).

Example:
```
use ::services::users;
```

**Comment by e-oz** [comment]
Just discovered simple paths like "use services::ItemUpdateError;" don't work also.

**Comment by matklad** [comment]
Most likely this is caused by switch to the new project model. Can you check the Cargo toolbar? Here are some more details about the toolbar: https://github.com/intellij-rust/intellij-rust/issues/1812#issuecomment-333780717

**Comment by e-oz** [comment]
@matklad yes, you were right.

For future readers: in Cargo toolbar (IDEA toolbar with Rust logo) click "+" and give it path to the Cargo.toml of your project. Not sure why it was so difficult for this toolbar to find existing Cargo.toml when this toolbar was introduced.
